File: motor3d 2.0.ConvertedToC#/ObjetoShading.cs
using Microsoft.VisualBasic;
using System;
using System.Collections;
using System.Collections.Generic;
using System.Data;
using System.Diagnostics;
using System.Linq;
using System.Xml.Linq;
namespace Motor3D.Escena.Shading
{
	public abstract class ObjetoShading : ObjetoEscena
	{

		public new event ModificadoEventHandler Modificado;
		public new delegate void ModificadoEventHandler(ref ObjetoEscena Sender);

		public ObjetoShading() : base()
		{
		}

		public override string ToString()
		{
			return base.ToString();
		}
	}
}

File: motor3d 2.0/objetoshading.vb
Namespace Escena.Shading
    Public MustInherit Class ObjetoShading
        Inherits ObjetoEscena

        Public Shadows Event Modificado(ByRef Sender As ObjetoEscena)

        Public Sub New()
            MyBase.New()
        End Sub

        Public Overrides Function ToString() As String
            Return MyBase.ToString()
        End Function
    End Class
End Namespace

